client/convert/protobuf_test.go

```diff
@@ -227,20 +227,6 @@ func TestConvert_Transaction(t *testing.T) {
 
 		assert.Equal(t, txA.ID(), txB.ID())
 	})
-
-	t.Run("With invalid arguments", func(t *testing.T) {
```

Comment:
this test no longer possible because we accept arbitrary bytes?

Reply:
Yep, Cadence encoding can't fail here anymore